lua/damagelogs/shared/events/equipment.lua

```diff
@@ -28,7 +28,7 @@ function event:TTTEquipmentUse(ply, equipment, info)
         [EVENT_DETAILS.UserName] = ply:Nick(),
         [EVENT_DETAILS.UserRole] = ply:GetRole(),
         [EVENT_DETAILS.UserSteamID] = ply:SteamID(),
-        [EVENT_DETAILS.EquipmentClass] = equipment:GetClass(),
+        [EVENT_DETAILS.EquipmentClass] = type(equipment) == "Weapon" and equipment:GetClass() or equipment,
```

Comment:
Is this to allow strings to be passed to this instead of entities?
It might be safer to do
`IsEntity(equipment) and equipment:GetClass() or equipment`

As it's possible that non-weapon entities could use this hook (e.g. planted bombs, custom NPCs, grenade entities)